### PR TITLE
avatar-badge

### PR DESCRIPTION
**Pull request by Sen-442b**
Avatar and Badge
**Feat:-**
    - Avatar with 4 different sizes 
    - Avatars with text 
    - Avatar with icon and text overlay
    - 3 variations of status badge 
    - 1 notification badge (on top of icons)
**Possible bugs: -**
    -May lead to duplicity of some styles after merging PR
    -Notification badge not responsive to different variations of icons 

**Incomplete feature: -**
-Dynamic border color based on the background color of parent container

**Bad Practices:-**
:- Redundant comments (kept them for upcoming features)

### CODE REVIEW

**Comment by Sen-442b** [comment]
image
image

**Review by Swapnilkota108 — APPROVED** [review]

**Comment by Swapnilkota108** [comment]
Good work bro!!!!

**Comment by Sen-442b** [comment]
> Good work bro!!!!

Thank you 🌟

File: avatar-and-badge/source/avatar.css [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+:root {
+  
+  --white-temp:rgb(255, 255, 255);
+  --online-status-color:rgb(59,165,93);
+  --dnd-status-color:#ed4245;
+  --offline-status-color:rgb(161, 159, 159);
+  --generic-transition:0.3s all ease-in;
+
+}
```
Comment (adidoshi) [inline avatar-and-badge/source/avatar.css:13]: Just a suggestion, if we could keep root css in the default css & then you can use it everywhere.

**Review by adidoshi — APPROVED** [review]
Components UI looks good.

File: avatar-and-badge/source/avatar.css (hunk 1 above)
Comment (Sen-442b) [reply avatar-and-badge/source/avatar.css:13]: Ohk, thanks for the suggestion🌟

**Pull request merged**